apis/pipeline/pipeline.yaml

```diff
@@ -2689,7 +2689,7 @@ paths:
                 default:
                     $ref: '#/components/responses/Error'
 
-    /api/v1/orgs/{orgId}/backups/{id}/sync:
+    /api/v1/orgs/{orgId}/clusters/{id}/backups/sync:
```

Comment:
**Note**: this is a public API breaking change which should be communicated accordingly and be updated in "internal" dependents. 
(Which IMO should also exclude the possibility of this change being released in a hotfix release.)

Not sure if this scope is the best to correct this API issue. 
Couldn't we leave the path alone or introduce the new path while leaving the original in place with a deprecation warning for a couple more releases before removing entirely (smoothing out the transition)?

Reply:
Yes, however this never worked. I mean the mapping was incorrect in OpenAPI spec only and as a result it was wrong in generated client, the actual backend path is not changed, just in OpenAPI spec.

Reply:
Well, that invalidates the second part of my comment I believe.

Reply:
Okay, now I have finally understood what you wrote, yeah, this is just a patch then.